rust/ql/lib/codeql/rust/internal/PathResolution.qll

```diff
@@ -604,7 +605,13 @@ private class EnumItemNode extends TypeItemNode instanceof Enum {
   }
 }
 
-private class VariantItemNode extends ItemNode instanceof Variant {
+/** An item that can be called with arguments. */
+abstract class CallableItemNode extends ItemNode {
```

Comment:
I think it's a bit confusing that `CallableItemNode` is a function or a struct, when we already have `Callable` which is a function or a closure. Could we name it something different? Maybe `HasParameters`?